maintenance/wiki2rss.php

```diff
@@ -100,8 +100,8 @@ function handleRevision( $rev ) {
 		 } 
 		}
 		
-		$html =  preg_replace("'<a href=\"http:\/\/(.*?)nofollow\">'si",'<a href="AYEXTHREFLINK">',$html); //replace links
-		$html =  preg_replace("'<a href=\"http:\/\/\"/index.php(.*?)\">'si",'<a href="AYINTHREFLINK">',$html); //replace links		
+		$html =  preg_replace("'<a href=\"https?:\/\/(.*?)nofollow\">'si",'<a href="AYEXTHREFLINK">',$html); //replace links
+		$html =  preg_replace("'<a href=\"https?:\/\/\"/index.php(.*?)\">'si",'<a href="AYINTHREFLINK">',$html); //replace links
 		$html =  preg_replace("'<script(.*?)</script>'si",'',$html); //replace links
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)

[project-id]: wikia:app